roles/ovirt-host-deploy-facts/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Fetch info about VDSM
+  command: bash -c "rpm -q vdsm --qf '%{VERSION}'"
```

Comment:
No need to make a call out to bash here.

Reply:
right, it was needed in grep+awk

Reply:
@machacekondra: so...you left it in the commit?

Reply:
Created #75 since this PR is already merged

Reply:
I did, because using `rpm` in `command` generates a warning in `command` task output. So I think it's better to have clean log.

Reply:
The warning says that you should use yum instead of executing rpm with command.

Reply:
you get that on which platform? Either way it sounds quite stupid so perhaps you can just silence it...

Reply:
@michalskrivanek it's a warning that Ansible spits out.  It's trying to discourage people from doing RPM/YUM installs directly but catches quite a few extra things in the process.   The fix is fairly simple once you know it exists:

https://github.com/oVirt/ovirt-ansible/pull/75/commits/49f9b5bbad61cb45c76bac3a575623a6f80939d6